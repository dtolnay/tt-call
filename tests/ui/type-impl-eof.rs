#[macro_use]
extern crate tt_call;

tt_call! {
    macro = [{ parse_type }]
    input = [{ impl }]
}

fn main() {}
