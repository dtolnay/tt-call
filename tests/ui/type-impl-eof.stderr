error: no rules expected the token `impl`
 --> tests/ui/type-impl-eof.rs:5:16
  |
5 |     input = [{ impl }]
  |                ^^^^ no rules expected this token in macro call
  |
  = note: while trying to match end of macro
